src/Phinx/Db/Adapter/SQLiteAdapter.php

```diff
@@ -40,7 +40,7 @@ class SQLiteAdapter extends PdoAdapter
     protected static array $supportedColumnTypes = [
         self::PHINX_TYPE_BIG_INTEGER => 'biginteger',
         self::PHINX_TYPE_BINARY => 'binary_blob',
-        self::PHINX_TYPE_BINARYUUID => 'binary_blob',
+        self::PHINX_TYPE_BINARYUUID => 'uuid_blob',
```

Comment:
What about binary(16)?

Reply:
Isnt Binary always 16?

Reply:
What would that change look like?

Reply:
WAH
I accidently pushed directly instead of creating a PR: https://github.com/cakephp/phinx/commit/5a78c766090c3abebb5b01778f96e53695781fcf

We can revert if needed!

Reply:
I checked, and it returns binary without length, as such this is not working out.
How can we also include the length I wonder.

Reply:
Now I understand: You mean literally `binary(16)`. Didnt know that works.